utils/mtpoe-ctrl/Makefile

```diff
@@ -0,0 +1,44 @@
+#
+# Copyright (C) 2018 adron@yapic.net
+#
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=mtpoe-ctrl
+PKG_SOURCE_VERSION:=9c42b74274cca10810363dbbb6d161bfb9cedb7e
+PKG_MIRROR_HASH:=1ac2902161ba4bb1e8af397aa48821ccea95740199b1c155f98161c7ff3a93df
+PKG_VERSION:=1.15
+PKG_RELEASE:=18
+
+PKG_SOURCE_PROTO:=git
+PKG_SOURCE_URL:=https://github.com/prudy/mtpoe_ctrl.git
+PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.gz
+PKG_SOURCE_SUBDIR:=$(PKG_NAME)-$(PKG_VERSION)
+
+PKG_MAINTAINER:=Sergey Sergeev <adron@yapic.net>
```

Comment:
Is @adron-s aware that he would be maintainer here?

Reply:
It's not a problem to put me there, and update when original author takes the PR. I kept that o  purpose, not to take credit for someone's work.

Reply:
Maintainership doesnt mean any credits other than you are the person that gets pinged on updates or issues.